build.gradle.kts

```diff
@@ -0,0 +1,54 @@
+import net.kyori.indra.IndraPlugin
+import net.kyori.indra.IndraCheckstylePlugin
+import net.kyori.indra.sonatypeSnapshots
+import org.checkerframework.gradle.plugin.CheckerFrameworkPlugin
+import org.checkerframework.gradle.plugin.CheckerFrameworkExtension
+
+plugins {
+  `maven-publish`
+  id("net.kyori.indra") version Versions.INDRA
```

Comment:
Unable to compile with java 8, setting my java version to 11 allows the plugin to compile.